### PR TITLE
Jest testing with examples.

### PR DESCRIPTION
**Pull request by illepic**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -26,7 +26,7 @@ before_script:
 - composer --version
 - node --version
 - npm --version
-script: npm run lint && npm run compile:pl
+script: npm run lint && npm run test:unit && npm run compile:pl
```
Comment (jryanconklin) [inline .travis.yml]: Nice! So we're including unit tests by default whereas pa11y is optional?

File: .eslintrc.js [hunk 2]
```diff
@@ -1,7 +1,7 @@
 // Rule reference: http://eslint.org/docs/rules
 // Individual rule reference: http://eslint.org/docs/rules/NAME-OF-RULE
 module.exports = {
-  extends: "airbnb",
+  extends: ["airbnb", "plugin:jest/recommended"],
```
Comment (jryanconklin) [inline .eslintrc.js]: It might be worth adding this to the comment above.

File: README.md [hunk 3]
```diff
@@ -563,7 +563,7 @@ npm install --save-dev pa11y@5.0.0-beta.5 pa11y-reporter-cli
 Then whenever you want to run your tests, simply:
 
 ```bash
-npm run pa11y
+npm run test:accessibility
```
Comment (jryanconklin) [inline README.md]: Simply, `test:a11y` might be a nice shorthand here for folks.

File: source/_patterns/01-atoms/button/__tests__/button.test.js [hunk 4]
```diff
@@ -0,0 +1,37 @@
+const $ = require('jquery');
```
Comment (jryanconklin) [inline source/_patterns/01-atoms/button/__tests__/button.test.js]: Tests are a part of our design system, and we don't expect these to move when / if demo folders and _twig-components move?

File: .travis.yml (hunk 1 above)
Comment (illepic) [reply .travis.yml]: pa11y current "fails". I'll get it included in Travis builds once it's passing.

File: source/_patterns/01-atoms/button/__tests__/button.test.js (hunk 4 above)
Comment (illepic) [reply source/_patterns/01-atoms/button/__tests__/button.test.js]: Tests stay close to components. Bundles get deployed. Won't be affected by demo reshuffling.

**Pull request merged**